base/options/options.html

```diff
@@ -132,7 +132,7 @@
 									<li>That's it!! Your image should appear in the preview box above.</li>
 									<li>Still confused? <a target="_blank" href="http://pratikabu.users.sourceforge.net/extensions/scrolltotop/base64help.html">Click here</a> for detailed explanation with pictures.</li>
 								</ol>
-								You can try on these sites to get amazing icons <a target="_blank" href="https://findicons.com">findicons.com</a>, <a target="_blank" href="http://www.iconarchive.com">iconarchive.com</a>, <a target="_blank" href="https://icons8.com">icons8.com</a>.
+								You can try on these sites to get amazing icons from <a target="_blank" href="https://findicons.com">findicons.com</a>, <a target="_blank" href="http://www.iconarchive.com">iconarchive.com</a>, <a target="_blank" href="https://icons8.com">icons8.com</a>.
```

Comment:
I think there are two places where the same text is used. One for Single icon and one for Dual icon. Do you want to change it at both the places and then create another pull req?